cmake/BLTOptions.cmake

```diff
@@ -80,8 +80,9 @@ endif()
 #------------------------------------------------------------------------------
 # Compiler Options
 #------------------------------------------------------------------------------
-option(ENABLE_ALL_WARNINGS       "Enables all compiler warnings on all build targets" ON)
-option(ENABLE_WARNINGS_AS_ERRORS "Enables treating compiler warnings as errors on all build targets" OFF)
+option(ENABLE_ALL_WARNINGS         "Enables all compiler warnings on all build targets" ON)
+option(ENABLE_WARNINGS_AS_ERRORS   "Enables treating compiler warnings as errors on all build targets" OFF)
+option(ENABLE_MSVC_STATIC_MD_TO_MT "For static linking with MS Visual Studio, enables changing /MD to /MT" ON)
```

Comment:
@kennyweiss suggested making this option dependent on BUILD_STATIC_LIBS.  I agree

Reply:
Also need to mark this option as advanced

Reply:
done